portal-web/docroot/html/portal/api/jsonws/index.jsp

```diff
@@ -38,22 +38,36 @@
 
 		<div id="content">
 			<div id="main-content">
-				<aui:row>
-					<aui:col cssClass="lfr-api-navigation" width="<%= 25 %>">
-						<liferay-util:include page="/html/portal/api/jsonws/actions.jsp" />
-					</aui:col>
-
-					<aui:col cssClass="lfr-api-details" width="<%= 75 %>">
-						<liferay-util:include page="/html/portal/api/jsonws/action.jsp" />
-					</aui:col>
-				</aui:row>
+				<div class="container-fluid">
+					<aui:row>
+						<aui:col cssClass="lfr-api-navigation p-3" width="<%= 25 %>">
+							<liferay-util:include page="/html/portal/api/jsonws/actions.jsp" />
+						</aui:col>
+
+						<aui:col cssClass="lfr-api-details p-3" width="<%= 75 %>">
+							<liferay-util:include page="/html/portal/api/jsonws/action.jsp" />
+						</aui:col>
+					</aui:row>
+				</div>
 			</div>
 		</div>
 
-		<footer id="footer" role="contentinfo">
-			<p class="powered-by">
+		<footer class="card fixed-bottom m-0 p-2 rounded-0" id="footer" role="contentinfo">
+			<p class="m-0 powered-by">
 				<liferay-ui:message key="powered-by" /> <a href="http://www.liferay.com" rel="external">Liferay</a>
 			</p>
 		</footer>
 	</div>
+
+	<script data-senna-track="permanent" id="APIScrollIntoView" type="text/javascript">
+		Liferay.once('endNavigate', function(event) {
+			setTimeout(function() {
+				const selected = document.querySelector('#services .lfr-api-signature.selected');
```

Comment:
Use of `const` here means that this can't be backported (it will fail linting on 7.3.x), so we might want to switch this to `var`.